test/resources/case17_policy_webhook/case17_invalid_remediation_policy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: case17-test-policy
```

Comment:
can you change this name more unique for validating remediation

Reply:
Yes I can, do you also want me to change the name of the other two policies as well?